README.md

```
@@ -9,6 +9,7 @@ Compared with [REditorSupport's R extension](https://marketplace.visualstudio.co
 > [!NOTE]
 > If you already have the REditorSupport (R) extension installed, or you're using Positron, Raven's R-console features (R console, plot viewer, data viewer) step aside by default — see [Coexistence](docs/coexistence.md). Raven still provides code intelligence and scope-aware help in either setup.
 
+
```

markdownlint-cli2 (0.22.1)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)